docs/FiltersSummary.md

```diff
@@ -40,6 +54,14 @@ If these filters do not meet your needs, you can also write your own filters.
 |-|-|-|
 | add_hyphens_date | Adds hyphens to a date without hyphens | `{{ PID.7.Value \| add_hyphens_date }}` |
 | format_as_date_time | Convert an YYYYMMDDHHmmssSSS string, e.g. 20040629175400000 to dateTime format, e.g. 2004-06-29T17:54:00.000z | `{{ PID.29.Value \| format_as_date_time }}` |
+| now | Provides current UTC time in "yyyy-MM-ddTHH:mm:ss.FFFZ" format | `{{ "" \| now }}` |
```

Comment:
These two filters have been refined to be more powerful in recent updates (PR #222, etc). Could you please help to update the contents for these two filters?

| format_as_date_time | Convert an YYYYMMDDHHmmssSSS string, e.g. 20040629175400000 to dateTime format, e.g. 2004-06-29T17:54:00.000z. A parameter could be set to handle time zone with "preserve", "utc" or "local". The default method is "local" | `{{ PID.29.Value \| format_as_date_time: 'utc' }}` |
| now | Provides current time in a specific format. The default format is "yyyy-MM-ddTHH:mm:ss.FFFZ" | `{{ '' \| now: 'dddd, dd MMMM yyyy HH:mm:ss' }}` |